CHANGES.md

```diff
@@ -1,3 +1,23 @@
+## v0.3.1
+
+Conduit 0.3.1 improves Conduit's resilience and transparency.
+
+* Proxy (data plane)
+  * The proxy now makes fewer changes to requests and responses being proxied. In particular, requests and responses without bodies or with empty bodies are better supported.
+  * HTTP/1 requests with different `Host` header fields are no longer sent on the same HTTP/1 connection even when those hostnames resolve to the same IP address.
+  * A connection leak during proxying of non-HTTP TCP connections was fixed.
+  * The proxy now handles unavailable services more gracefully by timing out while waiting for an endpoint to become available for the service.
+* Command-line interface
+  * `$KUBECONFIG` with multiple paths is now supported.
```

Comment:
Was this a non-Buoyant contribution? Would be nice to call out GitHub usernames if so.

Reply:
> Was this a non-Buoyant contribution? Would be nice to call out GitHub usernames if so.

There were two non-Buoyant contributions mentioned in the release notes. I added a reference to the PR and the author to each.